airflow/logging_config.py

```diff
@@ -73,7 +73,7 @@ def configure_logging():
 
     validate_logging_config(logging_config)
 
-    return logging_config
+    return logging_config, logging_class_path
```

Comment:
Thanks @KevinYang21 !

Is there any specific reason that we need to return `logging_config`?

Before this PR: `configure_logging()` is invoked in `airflow/settings.py`, but seems what it returns is not used.
After this PR: `logging_class_path` is returned by `configure_logging()`, then used in `airflow/utils/dag_processing.py`. But `logging_config` is still not used anywhere.

Kindly let me know if I missed anything. Thanks!

Reply:
@XD-DENG Thank you for reviewing. I actually don't think the return value is used anywhere. I was playing it safe by keep the original return value. After you mentioned, I think we can just remove that return value--we can always add it back if it is needed in the future.